specification/cognitiveservices/resource-manager/Microsoft.CognitiveServices/stable/2017-04-18/examples/CreateAccount.json

```diff
@@ -4,7 +4,7 @@
     "subscriptionId": "f9b96b36-1f5e-4021-8959-51527e26e6d3",
     "resourceGroupName": "felixwa-01",
     "accountName": "testCreate1",
-    "parameters": {
+    "account": {
```

Comment:
This can't be changed within an API version, but this is a non-compliant request body. Please see[ RPC](https://github.com/Azure/azure-resource-manager-rpc/blob/master/v1.0/resource-api-reference.md#put-resource).  The request body should be the same as the response body, minus any read-only properties. Location, kind, and sku are top-level properties.

Reply:
Hi Kirs, this is just a name in swagger.

The request body is a `CognitiveServicesAccount`, same as the response.